src/viewconfigs/horizontal-multivec-1.json

```
@@ -15,6 +15,13 @@
           2671196920.8697634,
           2671320441.9350777
         ],
+        "genomePositionSearchBoxVisible": true,
+        "genomePositionSearchBox": {
+          "autocompleteServer": "http://higlass.io/api/v1",
+          "chromInfoServer": "http://higlass.io/api/v1",
+          "visible": true,
+          "chromInfoId": "hg19"
```

```suggestion
          "chromInfoId": "hg38"
```
I think we want the hg38 assembly for all of these

---

Thanks for the suggestion.